function-repository/src/main/java/com/future/function/repository/feature/scoring/AssignmentRepository.java

```diff
@@ -17,7 +17,7 @@
       Batch batch, Long deadline, Pageable pageable
   );
 
-  Page<Assignment> findAllByBatchAndDeadlineGreaterThanOrderByDeadlineDesc(
+  Page<Assignment> findAllByBatchAndDeletedFalseAndDeadlineGreaterThanOrderByDeadlineDesc(
```

Comment:
Has this been indexed?

Reply:
cc @oliverrsebastian

Reply:
the deleted attribute is indexed, still considering whether create another migration for adding index or not since it can improve/worsen the performance